actionview/lib/action_view/helpers/asset_url_helper.rb

```diff
@@ -118,16 +118,67 @@ module Helpers
     module AssetUrlHelper
       URI_REGEXP = %r{^[-a-z]+://|^(?:cid|data):|^//}i
 
-      # Computes the path to asset in public directory. If :type
-      # options is set, a file extension will be appended and scoped
-      # to the corresponding public directory.
+      # This is the entry point for all assets.
+      # When using the asset pipeline (i.e. sprockets and sprockets-rails), the
+      # behavior is "enhanced". You can bypass the asset pipeline by passing in
+      # `public_folder: true` to the options.
```

Comment:
The RDoc parser doesn't understand back-ticks, you should rather use `<tt>public_folder: true</tt>`.